src/validator.rs

```diff
@@ -128,19 +163,29 @@ fn create_fingerprint_for_value(value: &Value) -> Result<Fingerprint, Error> {
     create_default_hash(value).map(|binary| binary.encode_hex())
 }
 
+#[allow(dead_code)]
 fn validate_signed_certificate(
     signed_certificate: &SignedCertificate,
+) -> Result<ValidatedCertificate, Error> {
+    validate_signed_certificate_selective(signed_certificate, ValidationChecks::All)
+}
+
+fn validate_signed_certificate_selective(
+    signed_certificate: &SignedCertificate,
+    validations: ValidationChecks,
```

Comment:
I need this param to use it here https://github.com/golemfactory/yagna/pull/2596/files#diff-20649a7c229dc2e44d79eabd324452fcf7d751bcd6d5cffafe08998636e15962R71

Reply:
Nvm, ValidationChecks bitflag has been removed.